galley/pkg/authplugins/none/none.go

```diff
@@ -22,7 +22,7 @@ import (
 	"istio.io/istio/galley/pkg/authplugin"
 )
 
-func returnAuth(map[string]string) ([]grpc.DialOption, error) { // nolint: unparam
+func returnAuth(_ map[string]string) ([]grpc.DialOption, error) { // nolint: unparam
```

Comment:
So, nolint declarations are not working anymore? Not for this particular case, but I think we want to have the ability to disable linter under certain, limited cases.